src/app/static/src/tests/integration/modelRun.itest.ts

```diff
@@ -73,4 +78,26 @@ describe("Model run actions", () => {
         expect(commit.mock.calls[0][0]["payload"].detail).toBe("Missing some results");
     });
 
+   it("can cancel model run", async () => {
```

Comment:
I don't think this test adds anything over and above the unit tests - may as well delete to save a few build seconds!